### ISSUE TITLE
Extract website link from string file for play store restriction dialog.

### ISSUE THREAD

**Issue by MohitMaliFtechiz** [kiwix/kiwix-android#3625]
**Describe the bug**
Extract the website link from the string file for the Play Store restriction dialog. Since it is not available to translate on translate wiki.

![Screenshot from 2023-12-21 18-52-45](https://github.com/kiwix/kiwix-android/assets/34593983/0796df16-e381-4237-a956-088181e1cd3e)

**Comment by kelson42** [comment]
I don’t understand this ticket.

**Comment by MohitMaliFtechiz** [comment]
@kelson42 On the upstream our website link is not available for translation in other languages. If the user checks the Help section in our application there will be no website link available for other languages like it is showing in the English version. And user does not know where to download the APK since there is no link showing.

![1703231185445](https://github.com/kiwix/kiwix-android/assets/34593983/1e234bfb-72b6-4744-962f-312f7db6e224)

**Comment by kelson42** [comment]
> On the upstream our website link is not available for translation in other languages.

I see, but what it is like this? Before fixing the problem, I would like to have a full explanation about the reasons behind it!

**Comment by MohitMaliFtechiz** [comment]
@kelson42 We had added the anchor tag in our English version for the website link(Where the user can download the APK) It is showing in the Help screen as shown in the above image. The User can click on that link and it takes the user to our website.

However anchor tags are not available to translate on translateWiki, so the website link will not show in the help screen in other languages of our application. It only shows the text to user without any website link so the user does not know where and from which website he can download the APK. So this behavior might confuse the user so to address this issue we have opened this ticket.

Now, we have extracted the website link from the string file to our coding part because we don't need to translate the link, and attaching this link in text while showing it to the user, as shown in the PR description.
